tldw_Server_API/app/core/Ingestion_Media_Processing/Audio/Parakeet_Core_Streaming/ws_server.py

```
@@ -0,0 +1,334 @@
+                            "type": "warning",
+                            "state": "fallback_not_applied",
+                            "message": "Whisper fallback not enabled in core route",
+                        })
+                # Optional live insights configuration
+                try:
+                    if LiveInsightSettings is not None and LiveMeetingInsights is not None:
+                        insights_payload = cfg.get("insights") or cfg.get("meeting_insights")
+                        if insights_payload is not None:
+                            try:
+                                insights_settings = LiveInsightSettings.from_client_payload(insights_payload)  # type: ignore[attr-defined]
+                            except Exception as insight_err:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)